test/models/user_test.rb

```diff
@@ -475,4 +475,84 @@ class UserTest < ActiveSupport::TestCase
     assert_raise(ActiveRecord::RecordNotFound) { collection.reload }
     assert_raise(ActiveRecord::RecordNotFound) { node.reload }
   end
+
+  # to run this test run the following command
```

Comment:
hm, this looks like a standard command to run a single test in rails, am i missing something? i don't think every test of the 1000+ we have needs an instruction on how to run it, it's not anything specific to this codebase.

Reply:
I added the command to make it easier for both myself and the developer who will be testing these cases. However, I will remove the commands to prioritize consistency.